lib/redmine/menu_manager/menu_helper.rb

```diff
@@ -118,7 +118,7 @@ def render_menu(menu, project=nil)
     menu_items_for(menu, project) do |node|
       links << render_menu_node(node, project)
     end
-    links.empty? ? nil : content_tag('ul', links.join("\n").html_safe, class: "menu_root")
+    links.empty? ? nil : content_tag('ul', links.join("\n").html_safe, class: "menu_root", id: "main-menu--content")
```

Comment:
Line is too long. [116/100]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.